ci.sh

```diff
@@ -42,18 +42,19 @@ update_version() {
     BUILD_VERSION=$VERSION_BUMP
 }
 
+install_prerequisites() {
+    sudo apt-get update
+    sudo apt-get install -y vim
```

Comment:
why is this needed? this is on the host running ci.

Reply:
This had me confused also.
https://github.com/aperture-data/aperturedb-python/blob/a913e41278ed0fe3b1181214a0a4e2c11104f3d1/ci.sh#L32

Do you know what command 'ex' is ? It's vim.